include/ck_tile/host/reference/reference_gemm.hpp

```diff
@@ -16,15 +16,15 @@ template <typename ADataType,
           typename AElementOp   = ck_tile::identity,
           typename BElementOp   = ck_tile::identity,
           typename ACCElementOp = ck_tile::identity>
-CK_TILE_HOST void reference_gemm(const HostTensor<ADataType>& a_m_k,
-                                 const HostTensor<BDataType>& b_n_k,
-                                 HostTensor<CDataType>& c_m_n,
+CK_TILE_HOST void reference_gemm(HostTensorView<const ADataType> a_m_k,
+                                 HostTensorView<const BDataType> b_n_k,
```

Comment:
Why do not extract HostTensorView<const ADataType> into template parameter, just like other reference

Reply:
because we are not calling these non-batched functions. so the modification cannot be well tested.